src/styles/blocks/catalog.scss

```
@@ -0,0 +1,22 @@
+.catalog {
+  margin: 0 auto;
+  padding: 50px 40px;
```

do u need horizontal padding here?why?u already have gap for it
```suggestion
  padding-top: 50px;
```